<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSubstanceWidget</class>
 <widget class="QWidget" name="CSubstanceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>32</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CSubstanceWidget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,1,0">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>+</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioBase">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Set as base substance</string>
     </property>
     <property name="statusTip">
      <string>Set as base substance</string>
     </property>
     <property name="whatsThis">
      <string>Set as base substance</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineCoefficient">
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Stoichiometric coefficient</string>
     </property>
     <property name="statusTip">
      <string>Stoichiometric coefficient</string>
     </property>
     <property name="whatsThis">
      <string>Stoichiometric coefficient</string>
     </property>
     <property name="text">
      <string>1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboMaterial">
     <property name="toolTip">
      <string>Substance</string>
     </property>
     <property name="statusTip">
      <string>Substance</string>
     </property>
     <property name="whatsThis">
      <string>Substance</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboPhase">
     <property name="toolTip">
      <string>Phase</string>
     </property>
     <property name="statusTip">
      <string>Phase</string>
     </property>
     <property name="whatsThis">
      <string>Phase</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
